### ISSUE TITLE
Allow canceling out all other notifications

### ISSUE THREAD

**Issue by codewizdevs** [caroso1222/notyf#145]
Hello,

i think its a good idea to add method to cancel all other notifications, seems useful with fast uis where users click a lot and they get spammed, i know i can handle it manually, but it would be nice to have native method.

**Comment by brgrz** [comment]
Agree, for instance allow three (configurable number) notifications to be pushed and after that 1) silently cancel others incoming and/or 2) if there are more incoming just show one another notyf saving "There are more notifications available.." (we actually have a notifications drawer in place for such cases)
